lib/internal_config.json

```diff
@@ -1,3 +1,4 @@
 {
-    "RETRY_DELAY_IF_RATE_LIMITED": 5000
+    "INITIAL_RETRY_DELAY_IF_RATE_LIMITED": 5000,
+    "MAX_RETRY_DELAY_IF_RATE_LIMITED": 600000
```

Comment:
How were these values chosen? I feel like a delay of 10 minutes is fairly long...

Reply:
They were chosen based on the expected value of 5 minutes when the client has backed off (10 minutes / 2 due to uniformly random distribution).